cmd/mysterium_client/command_run/command.go

```diff
@@ -3,16 +3,19 @@ package command_run
 import (
 	"encoding/json"
 	"errors"
+	"io"
```

Comment:
We should agree on code formatting guidelines.
Otherwise it will start to reformat it forward&backward.